enayah-frontend/src/app/globals.css

```
@@ -147,3 +149,203 @@
+  opacity: 0.55;
+}
+
+/* Previous/next month Sunday = muted red */
+.enayah-date-picker .rmdp-day span.enayah-other-month-day.enayah-sunday {
+  color: color-mix(
+    in oklab,
+    var(--destructive) 55%,
+    var(--muted-foreground)
+  ) !important;
+
+  opacity: 0.7;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)